image_client/client_tools.py

```diff
@@ -48,6 +48,8 @@ def parse_command_line():
 
     parser.add_argument('-d', '--date', nargs="?", help='date to use', default=None)
 
```

Comment:
adding argparser argument to allow optional -m flag to input md5 string to feed to PIC_undo_batch